tests/AuditHookMemberGroupSetTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace SilverStripe\Auditor\Tests;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use SilverStripe\Auditor\Tests\AuditHookTest\Logger;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\Session;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\MFA\Authenticator\LoginHandler;
+use SilverStripe\MFA\Authenticator\MemberAuthenticator;
+use SilverStripe\MFA\Method\MethodInterface;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Group;
+use SilverStripe\Security\Security;
+
+class AuditHookMemberGroupSetTest extends SapphireTest
```

Comment:
This file is almost identical to AuditHookManyManyListTest.php - delete one of these files and use a use a dataProvider to feed in some variables instead